Tests/SwiftParserTest/translated/TryTests.swift

```diff
@@ -125,71 +125,55 @@ final class TryTests: XCTestCase {
   func testTry11a() {
     AssertParse(
       """
-      try1️⃣ 2️⃣let singleLet = foo()
+      1️⃣try let singleLet = foo()
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: 'try' must be placed on the initial value expression, Fix-It replacements: 1 - 5 = '', 21 - 21 = 'try '
-        DiagnosticSpec(locationMarker: "1️⃣", message: "consecutive statements on a line must be separated by ';'"),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "expected expression in 'try' expression"),
-      ]
+        DiagnosticSpec(message: "'try' must be placed on the initial value expression", fixIts: ["move 'try' after '='"]),
+      ], fixedSource: "let singleLet = try foo()"
```

Comment:
What happens if there's already a `try`? IIUC we'd wrap it again, ie. end up with `try try foo()`?

Reply:
Good catch. I changed it to not add a second `try` and added a test case for it.